scripts/st2_install.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash -e
+
+# Input variables
+export ST2_DOCKER_SRC_PATH="/opt/opensds/orchestration"
+export ST2_WORKFLOW_SRC_PATH="/opt/opensds/orchestration/contrib/st2/opensds"
+export HOST_USER=demo_user
+export HOST_PASSWORD=demo_password
+export PACKS_PATH=/opt/stackstorm/packs/
+export HOST_IP=100.64.40.36
```

Comment:
Host IP will be static?

Reply:
Updated the script and Host IP is not required